src/Swarm/TUI/Controller.hs

```diff
@@ -454,7 +455,15 @@ saveScenarioInfoOnFinish = do
         ts <- use $ gameState . ticks
         let currentScenarioInfo :: Traversal' AppState ScenarioInfo
             currentScenarioInfo = gameState . scenarios . scenarioItemByPath p . _SISingle . _2
-        currentScenarioInfo %= updateScenarioInfoOnFinish t ts won
+
+        replHist <- use $ uiState . uiREPL . replHistory
+        let determinator = CodeSizeDeterminators initialCode $ replHist ^. replHasExecutedManualInput
+        currentScenarioInfo
+          %= updateScenarioInfoOnFinish
+            determinator
+            t
+            ts
+            won
```

Comment:
What a funny way to indent, I take it you put it on a new line and Fourmolu did the rest?

Reply:
Yes, it seems that putting the argument list on its own line causes Fourmolu to give a new lines to each individual argument.

But placing the newline just before `updateScenarioInfoOnFinish` results in a stable Fourmolu indentation on a single extra line.  I'll do that.